### ISSUE TITLE
Pretrained dumps as download

### ISSUE THREAD

**Issue by malteos** [wetneb/opentapioca#4]
Thanks for the amazing work! 

Can you provide pretrained dumps as download? That would make it much easier to test everything locally without the training step.

**Comment by wetneb** [comment]
Yes! I thought I had done that but now I remember that I got stuck by the file size limit on GitHub. I will see if I can upload that on Zenodo.

**Comment by wetneb** [comment]
I have added the PageRank vector and the bag of words model here: https://github.com/wetneb/opentapioca/releases/tag/v0.1.0
Those are the most expensive parts to train - indexing Wikidata in Solr should only take a few hours (if you restrict the types of the items as the sample profiles do).

**Comment by LiuYuLOL** [comment]
Hey, is it possible to provide the pre-trained classifier(s)? As far as I knew, with the provided two models, I still need a dataset-specific classifier. 

While I'm just interested in using it on some plain text, e..g. news articles, Wikipedia pages, etc. So could you provide one classifier, e.g.,  the one used in the current web API?

**Comment by wetneb** [comment]
Sure, I have added one to the release page.

**Comment by LiuYuLOL** [comment]
> Sure, I have added one to the release page.

Great! The model passed the test and is run with a web server at localhost:8457. Can I ask how to request an annotation to this server?

**Comment by eracle** [comment]
Thanks, I will include them in the docker build process! are there any other files that could speed up it?

At the moment I have the following steps that I would like to replace with a downloadable artefact:
 - Training BOW
 - Preprocessing
 - Creating Wikidata graph
 - Compiling Wikidata graph
 - Computing Pagerank

**Comment by senisioi** [comment]
@wetneb do you remember which sklearn version you used to pickle those models?
Loading `sample_classifier.pkl` is not possible with `sklearn==1.1.2`

**Comment by eracle** [comment]
if you check my branch there is the working version pinned:

https://github.com/eracle/opentapioca

btw
scikit-learn==0.20.3

**Comment by ziodave** [comment]
You can convert to sample classifier file to the latest sklearn 1.2 by running a couple of `bbe` (binary search/replace):

1. `bbe -e "s/sklearn.preprocessing.data/sklearn.preprocessing/g"`
2. `bbe -e "s/sklearn.svm.classes/sklearn.svm/g" sample_classifier~1.pkl`

**Comment by banyous** [comment]
Hi all,

it seems that these [files](https://github.com/opentapioca/opentapioca/releases/tag/v0.1.0) are no longer compatible with recent versions of scikit-learn (>1.3), and the solution @ziodave  didn't work for me. 

Is there a plan or a possibility for an updated pretrained files that are compatible with the recent versions of scikit-learn?

Thanks!

**Comment by wetneb** [comment]
I'm not working on this project at the moment but I would be happy to publish models if someone could train them :)

**Comment by weezymatt** [comment]
Hi Antonin! I've working with your software to test a few things with the spaCy wrapper. Would it be cool to share them (once training is done on my poor mac) with you? :)
